prove/Develop05/SimpleGoal.cs

```diff
@@ -11,10 +11,11 @@ public SimpleGoal(string name, string description, int points, string goalType )
         
     }
     
-    public override void RecordEvents()
+    public override int RecordEvents()
```

Comment:
the type of the returned variables, now I think that I am using polymorphism correctly because I have three classes with the same method but the way that we get the point in the checklist is different, as well as eternal goal because it will never get done.

Reply:
There is no polymorphism h ere because you are checking the type.  Polymorphism would be that when you call RecordEvent, the caller doesn't know what specific type of goal it has.